### DISCUSSION TITLE
version 8.0 is not found

### DISCUSSION THREAD

**Discussion by MohamedEssamTawab** [jasontaylordev/CleanArchitecture#1275] [Q&A] [answered]
Hi I am trying to use version 8.0 but its branch doesn't exist I tried to navigate through the link found in the documentation under the version section but the result is not found page can anyone help me please.

**Comment by jasontaylordev** [comment, marked as answer]
This is now fixed. Note that you can also use a specific version of the template. For example;

```bash
dotnet new install Clean.Architecture.Solution.Template::8.0.6
dotnet new ca-sln --help
```
